client.go

```diff
@@ -322,13 +322,13 @@ func (c *Client) GetRecentStats() ([]*Stats, error) {
 // Historical data is only available for prior months,
 // starting with January 2014.
 // If date IsZero(), returns the prior month's data.
-func (c *Client) GetHistoricalSummary(date time.Time) (*HistoricalSummary, error) {
+func (c *Client) GetHistoricalSummary(date time.Time) (*[]HistoricalSummary, error) {
 	req := &historicalSummaryRequest{}
 	if !date.IsZero() {
 		req.Date = date.Format("20060102")
 	}
 
-	var result *HistoricalSummary
+	var result *[]HistoricalSummary
```

Comment:
Good catch! For consistency with the other methods in the Client, can we return a slice of pointers (`[]*HistoricalSummary`) rather than a pointer-to-slice (`*[]HistoricalSummary`)?

Reply:
Yes I will do that

Reply:
this is done, anything else? @timpalpant